lib/template.html.erb

```diff
@@ -54,10 +54,9 @@
   <body>
     <h1><%= name %></h1>
     <div class="personal-info">
-      <% if email %><%= email %></br><% end %>
-      <% if location %><%= location %></br><% end %>
-      <% if address %><%= address %></br><% end %>
-      <% if homepage %><a href="<%= homepage %>"><%= homepage %></a><% end %>
+      <% SIDE_INFORMATION.each do |method| %>
+          <%= send(method) %></br>
```

Comment:
It does not check if particular attribute is present before printing out the </br>. Also homepage isn't formatted as a link.